packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -55,6 +55,10 @@ class AnalyzeCommand extends FlutterCommand {
         valueHelp: 'path-to-sdk',
         help: 'The path to the Dart SDK.',
         hide: !verboseHelp);
+    argParser.addOption('protocol-traffic-log',
+        valueHelp: 'path-to-protocol-traffic-log',
+        help: 'The path to write the request and response protocol.',
```

Comment:
nit: add something like "This is only intended to be used for debugging the tooling".